ui/src/components/card/QCard.sass

```
@@ -39,6 +39,9 @@
     &--vert
       padding: 16px
 
+  .q-card--dense &--vert
+    padding: 2px 16px
+
```

You can simplify it by doing this:
```suggestion
  &--dense &--vert
    padding: 2px 16px

```